README.md

```diff
@@ -294,6 +294,7 @@ If you haven't installed docker yet, you need to install it (example for linux d
 
 **Step 5**: Run molecule test
 
+    cd openwisp.openwisp2
```

Comment:
let's move this above, after the `git clone`

Reply:
Done, thanks a lot for your guiding! It's my first time to submit a PR to a open-source project. BTW, I notice it takes a lot time to run test (5 mins) when I was trying to `commit`, is it normal?

Reply:
Yes, it's normal for this repository.

Reply:
You do not need to add the build output in the commit message BTW, is that intentional?

Reply:
It's weird, I thought I didn't attach the build output in the commit message (maybe due to the wrong git command). Let me correct it anyway~

Reply:
Done, thanks!